src/App.tsx

```diff
@@ -15,7 +15,7 @@ polyfillCountryFlagEmojis();
 
 function App() {
   return (
-    <div className='bg-white text-gray-900 dark:bg-gray-900 dark:text-gray-100'>
+    <div className='bg-white text-gray-900 dark:bg-gray-900 dark:text-gray-100 h-screen'>
```

Comment:
looks like this caused some regression in the home page, it fills dark only for half - 
<img width="1735" height="648" alt="image" src="https://github.com/user-attachments/assets/b3370917-ebec-4890-a1ec-097f83f29ec9" />